rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/RocketChatWidgets.java

```diff
@@ -0,0 +1,26 @@
+package chat.rocket.android.widget;
+
+import android.content.Context;
+import com.facebook.drawee.backends.pipeline.DraweeConfig;
+import com.facebook.drawee.backends.pipeline.Fresco;
+import com.facebook.imagepipeline.backends.okhttp3.OkHttpImagePipelineConfigFactory;
+import com.facebook.imagepipeline.core.ImagePipelineConfig;
+
+import chat.rocket.android.widget.fresco.ImageFormatConfigurator;
+import okhttp3.OkHttpClient;
+
+public class RocketChatWidgets {
```

Comment:
I think this class name should include `Fresco` and be put in `fresco` package.

When another initialization logic is added in future, it would be better to define another class if it is not related to fresco.